constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -193,11 +194,9 @@ func (d *Driver) Query(ctx context.Context, target string, constraints []*unstru
 			return nil, fmt.Errorf("unknown constraint template validator: %s", constraint.GetKind())
 		}
 
-		assumeVAPEnforcementNotDisabled := assumeVAPEnforcementWithDefault(constraint, VAPDefaultYes)
```

Comment:
This is going away because VAP enforcement is merely another enforcement point?

Reply:
This is a check to see if we have created VAPB for this constaint or not. We depend on int-tree VAPB if it exists, else we use gk webhook for enforcement. 

You are correct, now we do not have `none` option to disable VAP for template/constraint combo, rather it would be through `scopedEnforcementAction` if user wants to disable it.

Reply:
SGTM